conda_smithy/templates/create_conda_build_artifacts.bat.tmpl

```diff
@@ -54,8 +54,8 @@ if defined BLD_ARTIFACT_PREFIX (
         echo ##vso[task.setVariable variable=BLD_ARTIFACT_PATH]!BLD_ARTIFACT_PATH!
     )
     if "%CI%" == "github_actions" (
-        echo ::set-output name=BLD_ARTIFACT_NAME::!BLD_ARTIFACT_NAME!
-        echo ::set-output name=BLD_ARTIFACT_PATH::!BLD_ARTIFACT_PATH!
+        echo BLD_ARTIFACT_NAME=!BLD_ARTIFACT_NAME! >> %GITHUB_OUTPUT%
+        echo BLD_ARTIFACT_PATH=!BLD_ARTIFACT_PATH! >> %GITHUB_OUTPUT%
```

Comment:
```suggestion
        echo BLD_ARTIFACT_NAME=!BLD_ARTIFACT_NAME!>> %GITHUB_OUTPUT%
        echo BLD_ARTIFACT_PATH=!BLD_ARTIFACT_PATH!>> %GITHUB_OUTPUT%
```

I think Windows might add an extra space otherwise.

Reply:
Yep, I think you're right.